src/scripts/gen-react-bindings.js

```diff
@@ -8,6 +9,38 @@ fs.mkdir(REACT_BINDINGS_DIRECTORY, { recursive: true })
 const COMPONENT_PREFIX = 'leo'
 const SVELTE_REACT_WRAPPER_PATH = '../shared/svelte-react.js'
 
+const getEventsTypeDefinition = (componentName, componentEventNames) => {
+  if (componentEventNames.length === 0) return null
+
+  return `
+export type ${componentName}EventProps = {
+${componentEventNames.map((event) => `  ${event}?: (event: CustomEvent) => void`).join(';\n')}
+}
+`
```

Comment:
Creates a type that gets associated with the component type during the generation of the react components.